core/src/Kokkos_Layout.hpp

```diff
@@ -52,6 +52,12 @@ struct LayoutLeft {
   using array_layout = LayoutLeft;
 
   size_t dimension[ARRAY_LAYOUT_MAX_RANK];
+  // we don't have a constructor to set this directly
+  // but we will deprecate the class anyway (or at least using an instance of
+  // this class) so I believe its acceptable to not introduce a new ctor and
+  // simply set this value directly in the internal functions where we construct
+  // instances of LayoutLeft
```

Comment:
```suggestion
  // we don't have a constructor to set the stride directly
  // but we will deprecate the class anyway (or at least using an instance of
  // this class) when switching the internal implementation to use mdspan
```
It's strange having a comment in the code base that says "I".

Reply:
done (in both places)